examples/hybrid_jobs/6_QNSPSA_optimizer_with_embedded_simulator/qnspsa_with_embedded_simulator.ipynb

```diff
@@ -0,0 +1,1100 @@
+    "\n",
+    "We recommend to perform the benchmarking tests with Braket Jobs using the embedded simulator. When compared to the Braket notebooks, Braket Jobs has a few advantages for this workload:\n",
+    "\n",
+    "- First, we would like our benchmarking tests to be reproducible, which requires a systematic way to track the logs and the outputs. This is easily done with Braket Jobs. With the notebook, we would run into connection/permission timeout issues when the execution time of our program goes up to a few hours. We could easily lose the returned results or printed logs due to this issue. \n",
+    "\n",
+    "- Second, for tests on a larger scale with larger quantum circuits, we will likely need larger instances, with more abundant memory and computing power. For those more expensive instances, to be frugal, we would like to be charge only for the hours when the code is running. We would not be able to turn off the notebook instance at 3 am right after its execution, but Braket Jobs automatically do so to the Job instances."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "21aafdf1",
+   "metadata": {},
+   "source": [
+    "Yet we will start with creating a working sample in the Braket Jobs local mode (`LocalQuantumJob`, with its corresponding [API reference](https://amazon-braket-sdk-python.readthedocs.io/en/latest/_apidoc/braket.jobs.local.local_job.html) and [an example](https://github.com/aws/amazon-braket-examples/blob/main/examples/hybrid_jobs/4_Embedded_simulators_in_Braket_Jobs/Embedded_simulators_in_Braket_Jobs.ipynb) under section \"Debug with local mode\"), instead of directly proceeding to execute it remotely (`AwsQuantumJob`). Braket service needs to prepare the environment (Docker container) for each remote Job one submit, and this process typically takes up to a few minutes. On the other hand, with the local mode, the environment only needs to be prepared once during our first call. We can fail fast and iterate with the local mode, and move to the remote Jobs after debugging to better utilize its power."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "1cdd1724",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "292282985366.dkr.ecr.us-east-1.amazonaws.com/amazon-braket-base-jobs:1.0-cpu-py37-ubuntu18.04\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Retrieve one of the default images (environment), with pennylane pre-installed\n",
+    "region_name = \"us-east-1\" # Change the region as needed\n",
+    "braket_client = boto3.client(\"braket\", region_name=region_name)\n",
+    "aws_session = AwsSession(braket_client=braket_client)\n",
```

Comment:
I don't think this needs to be specified as long as you have the default region set in awscli?